pupil-spa/src/app/services/app-count/app-count.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AppCountService {
```

Comment:
👍

Reply:
@jon-shipley I thought it as well afterwards that reset was completely pointless.
Removed